openpype/hosts/maya/plugins/publish/collect_look.py

```diff
@@ -165,7 +170,7 @@ def get_file_node_path(node):
         if any(pattern in lower for pattern in patterns):
             return texture_pattern
 
-    if cmds.nodeType(node) == 'aiImage':
+    if cmds.nodeType(node) in ['aiImage', 'PxrTexture']:
```

Comment:
Would lookup in `set` be better?
```suggestion
    if cmds.nodeType(node) in {'aiImage', 'PxrTexture'}:
```